docs-v2/understand/concepts/events.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Event-based scripts'
+sidebarTitle: 'Events'
+description: 'Learn more about event-based scripts.'
+---
+
+# Overview
+Nango events are specific occurrences within the Nango system that can trigger custom actions. These events allow you to execute integration scripts in response to particular situations, enabling more dynamic and responsive integrations.
+
+# Types of Nango Events
+
+Nango currently supports the following events:
+- `post-connection-creation` occurs when a connection has been created.
+- `pre-connection-deletion` occurs before a connection is deleted.
+
+# Use cases
+Event-based scripts are particularly useful for:
+1. __Connection setup__. Ex:
+    - Setting up webhooks with the external service.
+    - Creating default resources or configurations.
+2. __Cleanup operations__. Ex:
+    - Removing webhooks from external services.
+    - Notifying external systems about the connection termination.
+    - Cleaning up associated resources.
+
+# Main characteristics
+
+## Trigger
+Event-based Scripts are __automatically__ triggered by Nango:
+    - They execute immediately when the event occurs.
+    - No manual intervention is required.
+    - They cannot be scheduled or delayed.
+    - Events are triggered exactly once per occurrence.
+    - Their executions are logged and can be monitored.
+
+## Constraints
+Event-based scripts operate under specific __constraints__:
+    - They cannot save, update or delete records.
+    - They focus on side effects and external system interactions.
+    - They can make API calls to external systems.
+    - They can read existing connection data.
+    - They can log information for monitoring.
+    - They can modify external resource states.
```

Comment:
I wonder if a table would be clearer

Reply:
can you think of a place in the docs where we use a table to format similar info?

Reply:
can you think of a place in the docs where we use a table to format similar info?